lib/mongoose_push_web/schemas/request/send_notification/deep/alert.ex

```diff
@@ -0,0 +1,34 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep.Alert do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Request.SendNotification.Deep.Alert",
+    description: "In this request one can pass alert field only.",
+    type: :object,
+    allOf: [MongoosePushWeb.Schemas.Request.SendNotification.Deep],
+    properties: %{
+      alert: %Schema{
+        type: :object,
+        properties: %{
+          body: %Schema{type: :string, description: "Body of the notification", format: :string},
```

Comment:
What is `format` here?

Reply:
I'm not sure I got your point there. Pure string format, w/o any limitations or regexps, as spotted in other places of this PR.

Reply:
Ok, but `format` value, seems to be more a guidline on how the value should look like. In this, regard, show does `format: :string` help? We already know that the `type` is `:string`, so this feels redundant. OpenAPI spec lists the following examples: `email`, `uuid`, `uri`,
`hostname` etc. If anything, I would suggest setting `format: :text` to indicate that this is non-binary value, but `string` here is too generic.

Reply:
Sounds ok to me. So if you don't mind I'll change it to `format: :text` also regarding `description`, `body`, `title` and `reason` fields throughout the whole PR, wherever I introduced `:string`.